docs/core/tools/telemetry.md

```diff
@@ -42,16 +42,14 @@ The .NET Core CLI Tools telemetry feature is enabled by default. Opt-out of the
 
 The feature collects the following data:
 
-- Timestamp&#8224;
-- Command (for example, "build")&#8224;
+- Timestamp of invocation&#8224;
```

Comment:
fyi: I removed the word merely on localization concerns ... and kind'a because it's obvious without the "of invocation" part.